### ISSUE TITLE
Whitepaper describing the Typelevel ecosystem

### ISSUE THREAD

**Issue by jducoeur** [typelevel/governance#50]
(Potentially controversial idea, here for tracking purposes.)

A recent conversation on Discord got into the topic of what sort of marketing we can and should be doing.  The most basic table stakes here is a whitepaper: a concise document, preferably just a few pages long, that lays out the Typelevel ecosystem from a ten thousand foot perspective, and how the various pieces come together to build systems.  It should be focused on high-level decision makers who are trying to understand the platform options for their new application, rather than the in-the-trenches programmers who we usually focus on.

**Comment by reardonj** [comment]
@jducoeur , I don't suppose this ever materialized somewhere?

**Comment by jducoeur** [comment]
No, it fell prey to the Thousand Projects Problem.  It would still be useful, I suspect, if anybody has the cycles to spearhead it, but I've clearly failed to do so.
